apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -3416,14 +3412,14 @@ a.btn.btn-xs.btn-link.pull-right {
     align-items: center;
 }
 
-.stock-info {
+.stock_info {
     position: absolute;
     top: 0;
     bottom: 0;
     right: 0;
 }
 
-.stock-wrap {
+.stock_wrap {
```

Comment:
This change is reverted while according to the code design, we've fixed most of the camel case classes. @prateek-giddh

Reply:
Done